sa/storage-authority.go

```diff
@@ -168,12 +168,18 @@ func (ssa *SQLStorageAuthority) GetAuthorization(id string) (authz core.Authoriz
 		}
 		authD := authObj.(*authzModel)
 		authz = authD.Authorization
+	} else {
+		authD := *authObj.(*pendingauthzModel)
```

Comment:
Swap the else and the condition check here. It's a little scary to have this code so far away from the original type when it will panic if it goes wrong.

Maybe even break the authzModel half of this branch out to a method to prevent accidents and confusion around the reuse of authObj.